src/test/resources/Features/verifyDataInCriteriaDetails.feature

```diff
@@ -1,4 +1,5 @@
 @Regression
+@criteriaDetails
```

Comment:
What's this for?

Reply:
It's much easier to run specific group of tests on Jenkins using tags, so added this one to execute tests that validate criteria details